tests/foreman/api/test_lifecycleenvironment.py

```diff
@@ -198,3 +198,26 @@ def test_positive_search_in_org(self):
             {lc_env_.name for lc_env_ in lc_envs},
             {u'Library', lc_env.name},
         )
+
+    @tier2
+    @skip_if_bug_open('bugzilla', 1348727)
+    def test_verify_bugzilla_1348727(self):
```

Comment:
FWIW I recommended earlier on doing something like this: https://github.com/SatelliteQE/robottelo/commit/eb4f18438727fd0c67999a00a4327fbac70b4030#diff-a7f0e38029470d303cc742786a7b2e3bL888

In short:
1. Test case names need not use bugzilla numbers
2. Decorate the test with `@skip_if_bug_open`
3. Add `@bz` docstring token to show the bugzilla number: like `@bz: 1222118`
4. In addition to 2&3, the first line of the test may show bugzilla numbers for easy reference like:
    `# see BZ #1222118`
5. Once the bugzilla is fixed, items 2 and 3 above can be removed.  Item 4 above will still be in place which will help referring the bug in feature if the test fails again.